ipn/ipnlocal/local.go

```diff
@@ -1833,7 +1833,10 @@ func (b *LocalBackend) checkPrefsLocked(p *ipn.Prefs) error {
 	if p.RunSSH {
 		switch runtime.GOOS {
 		case "linux":
-			// okay
+			if distro.Get() == distro.Synology && !envknob.UseWIPCode() {
```

Comment:
I don't know what `!envknob.UseWIPCode()` does but assume it is appropriate here.

Reply:
It lets adventurous people (perhaps building it themselves and running as root or in Docker, etc) turn on our general I-know-what-I'm-doing env knob to let it run anyway. Same as darwin support a few lines down.